app/main/posts/modify/post-media.directive.js

```diff
@@ -16,15 +16,27 @@ function (
     return {
         restrict: 'E',
         replace: true,
-        require: '^^form',
+        require: ['^^form', 'ngModel'],
```

Comment:
Take a look at how we handle this in the location directive. You really shouldn't need to require the form controller, just ngModel